route/route.go

```diff
@@ -501,22 +501,60 @@ func (r *Router) Export(ctx context.Context, req *collectortrace.ExportTraceServ
 					if sevent.Attributes != nil {
 						addAttributesToMap(attrs, sevent.Attributes)
 					}
+					for k, v := range resourceAttrs {
+						attrs[k] = v
+					}
 					sampleRate, err := getSampleRateFromAttributes(attrs)
 					if err != nil {
 						debugLog.
 							WithField("error", err.Error()).
 							WithField("sampleRate", attrs["sampleRate"]).
 							Logf("error parsing sampleRate")
 					}
-					events[i+1] = &types.Event{
+					events = append(events, &types.Event{
 						Context:    ctx,
 						APIHost:    apiHost,
 						APIKey:     apiKey,
 						Dataset:    dataset,
 						SampleRate: uint(sampleRate),
 						Timestamp:  timestamp,
 						Data:       attrs,
+					})
+				}
+
+				for _, slink := range span.Links {
+					var timestamp time.Time
```

Comment:
It doesn't look like this is getting set anywhere?

Reply:
I copied this from Shepeherd. Span links don't have timestamps, so instead we should just omit the `timestamp` event field.